data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -1747,12 +1747,13 @@ Figurine of Wondrous Power (Ivory Goats)											OUTPUTNAME:Figurine of Wondro
 Figurine of Wondrous Power (Onyx Dog)											OUTPUTNAME:Figurine of Wondrous Power, [NAME]								TYPE:Magic.Wondrous																										COST:15500	WT:0																		SOURCEPAGE:p.257																																																																																																																																																																																																																																																																SPROP:This figurine of wondrous power appears to be a miniature statuette of a dog an inch or so high. When the figurine is tossed down and the correct command word spoken, it becomes a living creature of normal size. The creature obeys and serves its owner. The creature understands Common but does not speak. If the figurine is broken or destroyed in its statuette form, it is forever ruined. All magic is lost, its power departed. If slain in animal form, the figurine simply reverts to a statuette that can be used again at a later time. When commanded, this statuette changes into a creature with the same properties as a riding dog (see page 272 of the Monster Manual), except that it is endowed with an Intelligence of 8, can communicate in Common, and has exceptional olfactory and visual abilities. (It has the scent ability and adds +4 to its Spot and Search checks.) It has 60-foot darkvision, and it can see invisibility. An onyx dog can be used once per week for up to 6 hours. It obeys only its owner. Moderate transmutation; CL 11th
+Figurine of Wondrous Power (Onyx Dog).MOD	SPROP:Riding Dog:Hit Dice: 2d8+4 (13 hp),Initiative: +2, Speed: 40 ft, Armor Class: 16 (+2 Dex, +4 natural), touch 12, flat-footed 14, Base Attack/Grapple: +1/+3, Attack: Bite +3 melee (1d6+3), Full Attack: Bite +3 melee (1d6+3), Space/Reach: 5 ft./5 ft. Special Qualities: Low-light vision, scent, Saves: Fort +5, Ref +5, Will +1,  Abilities: Str 15, Dex 15, Con 15, Int 2, Wis 12, Cha 6,  Skills: Jump +8, Listen +5, Spot +5, Swim +3, Survival +1, Feats: Alertness, Track. Carrying Capacity: light 100, medium load 101–200, ; and a heavy load 201–300 pounds. A riding dog can drag 1,500 pounds.	PRERULE:1,DisplayFullEquip
```

Comment:
I don't think this is going to have the desired outcome. The PRERULE is not attached to anything.

Reply:
without the pipe it's disabling the entire MOD line, thus not adding the SPROP to the default SPROP -